src/structures/Guild.js

```diff
@@ -484,6 +486,15 @@ class Guild extends Base {
     return new Date(this.joinedTimestamp);
   }
 
+  /**
+   * Whether Membership Screening is enabled on this guild
+   * @type {boolean}
+   * @readonly
+   */
+  get membershipScreeningEnabled() {
+    return this.features.includes('MEMBER_VERIFICATION_GATE_ENABLED');
+  }
+
```

Comment:
this is not needed imo; .partnered and .verified were hacks on checking other features (added before the PARTNERED and VERIFIED features iirc) and should be removed too in a separate pr; `guild.membershipScreeningEnabled` is barely an improvement over `guild.features.has('MEMBER_VERIFICATION_GATE_ENABLED')`

Reply:
yeah, i see your point, especially for v13 where we are trying to remove these simple wrappers, will remove